vision_datasets/common/dataset_info.py

```diff
@@ -83,3 +88,26 @@ def __init__(self, dataset_info_dict):
     @property
     def task_names(self):
         return list(self.sub_task_infos.keys())
+
+
+class KVPairDatasetInfo(DatasetInfo):
+    def __init__(self, dataset_info_dict):
+        data_type = _data_type_to_enum(dataset_info_dict.get('type'))
```

Comment:
Is this line still necessary if L104 will still `__init__` with the parent class, which does this?